llvm_propeller_profile_writer.cc

```diff
@@ -82,7 +82,7 @@ void DumpCfgs(
 namespace devtools_crosstool_autofdo {
 
 using ::devtools_crosstool_autofdo::PropellerOptions;
-using ::llvm::Optional;
```

Comment:
`Optional` is also used below (line 255 and more).

I would suggest removing `using` here and replacing `Optional` with `std::optional` below.

Reply:
`Optional` at line [L255](https://github.com/google/autofdo/blob/master/llvm_propeller_profile_writer.cc#L255) has to be replaced with `std::optional`.
Same in [L298](https://github.com/google/autofdo/blob/master/llvm_propeller_profile_writer.cc#L298) and [L300](https://github.com/google/autofdo/blob/master/llvm_propeller_profile_writer.cc#L300).

Reply:
Fixed, thanks!!